src/multipleworld/world_factory.py

```diff
@@ -0,0 +1,105 @@
+from typing import Dict, List, Optional
+from pycram.datastructures.enums import WorldMode
+from pycram.world import World
+
+
+class WorldFactory:
```

Comment:
I think the WorlFactory should be a singelton, since we only want exactly one. 
See here: https://python-patterns.guide/gang-of-four/singleton/ or here: https://github.com/cram2/pycram/blob/dev/src/pycram/local_transformer.py#L148 for implementation details